### PR TITLE
Data and interactive map changes 

### PR DESCRIPTION
**Pull request by wtruitt3**
Adds all the new bike data for each city and adds DC, Chicago, and Philadelphia to the map.
Updated About section
Added functionality for a border URL param

### CODE REVIEW

File: data/philly-bike/indego-stations-2022-04-01.csv [hunk 1]
```diff
@@ -0,0 +1,190 @@
+Station_ID,Station_Name,Day of Go_live_date,Status
```
Comment (aberke) [inline data/philly-bike/indego-stations-2022-04-01.csv:1]: what is this file? Is this file needed?

**Comment by aberke** [comment]
What happens with the bike data files for dc and chicago once they're downloaded? Do you delete them? Consider using a .gitignore + .keep file (see the boston-bike directory)

File: data/dc/dc_limits copy.geojson [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{
```
Comment (aberke) [inline data/dc/dc_limits copy.geojson:1]: Should this file be here? It is copied from the other dc_limits file. Keep just one or comment why there must be two.

File: data/il/chicago_limits copy.geojson [hunk 3]
```diff
@@ -0,0 +1,244 @@
+{
```
Comment (aberke) [inline data/il/chicago_limits copy.geojson:1]: same comment as with the dc_limits copy file.

File: viz/index.js [hunk 4]
```diff
@@ -1,142 +1,171 @@
...
+};
 
+const bikeCheck = document.getElementById("b-bikes");
+const incomeCheck = document.getElementById("b-income");
+const raceCheck = document.getElementById("b-race");
+let borderCheck = false;
+bikeCheck.addEventListener("click", toggleBikeStationLayer);
+incomeCheck.addEventListener("click", toggleIncomeLayer);
+raceCheck.addEventListener("click", toggleRaceLayer);
+// borderCheck.addEventListener("click", toggleBorderLayer);
```
Comment (aberke) [inline viz/index.js:164]: remove commented out code or add a comment about why it is commented out for now

File: viz/index.js [hunk 5]
```diff
@@ -145,73 +174,72 @@ const hideLoadingScreen = () => {
...
-const URL_PARAM_YEAR = 'year';
-const URL_PARAM_INCOME = 'i';
-const URL_PARAM_RACE = 'r';
-const URL_PARAM_BIKES = 'b';
+const URL_PARAM_CITY = "city";
+const URL_PARAM_YEAR = "year";
+const URL_PARAM_INCOME = "i";
+const URL_PARAM_RACE = "r";
+const URL_PARAM_BIKES = "b";
+const URL_PARAM_BORDER = "l";
```
Comment (aberke) [inline viz/index.js]: I suggest making this a more meaningfully named parameter.
e.g. "study_boundary"
const URL_PARAM_STUDY_BOUNDARY = "study-boundary"
and then add documentation for how to activate this feature in the README (see documentation for ?ha)

File: viz/index.js [hunk 6]
```diff
@@ -145,73 +174,72 @@ const hideLoadingScreen = () => {
...
   let r = url.searchParams.get(URL_PARAM_RACE);
   let b = url.searchParams.get(URL_PARAM_BIKES);
-  if (i == null)
-    incomeCheck.checked = true;
-  if (r == null)
-    raceCheck.checked = true;
-  if (b == null)
-    bikeCheck.checked = true;
-  updateYearHTML(year || '');
+  // let l = url.searchParams.get(URL_PARAM_BORDER);
```
Comment (aberke) [inline viz/index.js]: why is this commented  out?

File: viz/index.js [hunk 7]
```diff
@@ -145,73 +174,72 @@ const hideLoadingScreen = () => {
...
+  if (i == null) incomeCheck.checked = true;
+  if (r == null) raceCheck.checked = true;
+  if (b == null) bikeCheck.checked = true;
+  // if (l == null) borderCheck.checked = true;
```
Comment (aberke) [inline viz/index.js]: why is this commented out?

**Comment by aberke** [comment]
Overall great work. There are a few changes to make.
Please see in the inline comments.
It is a little hard to see the diffs and evaluate the changes to the .html and .js files because of the formatting changes.
Please rename the "l" param to "study-boundary" (see inline comment) and document in the README.

In a following commit, please update the text in the map html.
See notes for August Week 2:
Remove much of the text below the map which will later be replaced with a link to the paper. Probably do: 
- remove the 2 last text paragraphs, keep About, About the Data, footer
- update About the Data to remove the notes about dates, and remove "​​For more about the data scraping and processing, see [code and documentation](https://github.com/aberke/income-race-bikes)"

File: data/philly-bike/indego-stations-2022-04-01.csv (hunk 1 above)
Comment (wtruitt3) [reply data/philly-bike/indego-stations-2022-04-01.csv:1]: yes this file is needed because the bike data for philly doesn't have names so I go through this file to add names to the stations

File: data/dc/dc_limits copy.geojson (hunk 2 above)
Comment (wtruitt3) [reply data/dc/dc_limits copy.geojson:1]: No these files shouldn't be there they don't show up in my repository so I don't know why they're in the pull request there should just be one

**Comment by wtruitt3** [comment]
I will change the param and update the readme, and I'm pretty sure I changed the about section already - I can double check on that. Also will add a git ignore to the other bike data files.

File: data/philly-bike/indego-stations-2022-04-01.csv (hunk 1 above)
Comment (aberke) [reply data/philly-bike/indego-stations-2022-04-01.csv:1]: You should probably add a README to the data directory (`data/README.md`) explaining this and why the bike data is kept for philly.
The data/README.md file can also detail how data is handled overall vs treated differently for the different areas.

File: README.md [hunk 8]
```diff
@@ -10,41 +10,34 @@ The map explores this question visually.
```
Comment (aberke) [inline README.md:10]: great work on this

File: data/README.md [hunk 9]
```diff
@@ -0,0 +1,22 @@
+# Specifics about bike-share data scraping
```
Comment (aberke) [inline data/README.md:1]: perfect
